test/WebJobs.Extensions.RabbitMQ.Samples/RabbitMQSamples.cs

```diff
@@ -77,7 +77,16 @@ public static class RabbitMQSamples
         // Trigger samples
         // Defaults to localhost if HostName is not specified and connection string is not set in appsettings.json
         public static void RabbitMQTrigger_String(
-             [RabbitMQTrigger("queue")] string message,
+             [RabbitMQTrigger(connectionStringSetting: "rabbitMQTest", "queue")] string message,
+             string consumerTag,
+             ILogger logger)
+        {
+            logger.LogInformation($"RabbitMQ queue trigger function processed message consumer tag: {consumerTag}");
+            logger.LogInformation($"RabbitMQ queue trigger function processed message: {message}");
+        }
+
+        public static void RabbitMQTrigger_String_NoConnectionString(
+             [RabbitMQTrigger(hostName: "localhost", userNameSetting: "%UserNameSetting%", passwordSetting: "%PasswordSetting%", port: 5672, queueName: "queue")] string message,

```

Comment:
rename to "localhost" to "RabbitMQHostName" to avoid confusion of specifying user name / pwd for localhost